src/styles/tools/index.css

```diff
@@ -1,4 +1,5 @@
 @import 'ec-alert';
+@import 'ec-dropdown';
```

Comment:
We should remove the ec-dropdown.scss from scss folder after it was moved to ec-dropdown.css.

Reply:
I actually cannot find where these mixins are used except the ec-dropdown-search.vue. I'd swear these mixins should be used in EBO, but I cannot find it...

Reply:
- [ ] Remove unused scss

Reply:
@mcibique I couldn't find any reference in EBO neither. I only see it is used in the ec-dropdown-search.vue

Reply:
I found them, ebo-supplier-dropdown was using the sass mixin, but during EBO TW refactoring, these were removed and now the dropdown has a visual bug. I'll create a story for fixing that and use the mixins you have just migrated in it again.